code/workspaces/web-app/src/version.js

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 
-let version = 'development';
+let version;
```

Comment:
Does this still work for locally running instances?

Reply:
Yes as the local json file has development in it.